spec/lib/newspaper_works/ingest/ndnp/issue_metadata_spec.rb

```diff
@@ -55,6 +62,19 @@
       expect(issue.publication_date).to eq "1857-02-14"
     end
 
+    it "gets publication title via label, when reel unavailable" do
+      expect(issue.publication_title).to \
+        eq 'Weekly Trinity journal (Weaverville, Calif.)'
+    end
+
+    # integration test for reel context publication title:
+    it "gets publication title via label, from reel" do
+      expect(issue_ingest.metadata.publication_title).to \
+        eq 'Weekly Trinity journal (Weaverville, Calif.)'
+      expect(issue_ingest.metadata.publication_title).to \
```

Comment:
Just curious, is Rubocop mandating the `\` syntax for multi-line `expect` statements? I've not seen this before...

Reply:
Yes, for some reason, rubocop was unhappy with second-line indentation without it.